{"text": "@Worldometers Hi @Worldometers, I've been closely following the COVID-19 section of the website. Great effort! I saw yesterday that you have updated the # of tests carried out in Brazil, but I couldn't find the source you've used. Could you please refer me to it? Thank you.", "in_reply_to_user_id": "345301111", "author_id": "2861694131", "lang": "en", "created_at": "2020-04-22T02:14:35.000Z", "referenced_tweets": [{"type": "replied_to", "id": "1246336010784563201"}], "possibly_sensitive": false, "id": "1252782825599729665", "context_annotations": [{"domain": {"id": "123", "name": "Ongoing News Story", "description": "Ongoing News Stories like 'Brexit'"}, "entity": {"id": "1220701888179359745", "name": "COVID-19"}}, {"domain": {"id": "29", "name": "Events [Entity Service]", "description": "Entity Service related Events domain"}, "entity": {"id": "1219057585707315201", "name": "COVID-19: Latest news updates from around the world", "description": "This page is a timeline of Tweets with the latest information and updates on COVID-19 from experts, journalists, health authorities and other trusted sources from around the world. For more on COVID-19, visit: twitter.com/explore/tabs/covid-19"}}], "reply_settings": "everyone", "entities": {"annotations": [{"start": 179, "end": 184, "probability": 0.9993, "type": "Place", "normalized_text": "Brazil"}], "mentions": [{"start": 0, "end": 13, "username": "Worldometers", "id": "345301111"}, {"start": 17, "end": 30, "username": "Worldometers", "id": "345301111"}]}, "source": "Twitter Web App", "public_metrics": {"retweet_count": 0, "reply_count": 0, "like_count": 0, "quote_count": 0}, "conversation_id": "1246336010784563201"}